docs/pages/guides/local-app-production.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Create a local production build locally
+sidebar_title: Production
+description: Learn how to create a production build for your Expo app locally.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+To create your app's production build (also known as release build) locally, you need to follow separate steps on your computer and use the tools required to create any native app. This guide provides the necessary steps to create a production build for Android and iOS locally.
+
+## Android
+
+Creating a local production build for Android requires signing it with an [upload key](https://developer.android.com/studio/publish/app-signing#certificates-keystores) and generating an Android Application Bundle (**.aab**). Follow the steps below to create a production build:
+
+### Prerequisites
+
+- [OpenJDK distribution](/get-started/set-up-your-environment/?mode=development-build&buildEnv=local#install-watchman-and-jdk) installed to access the `keytool` command
+- **android** directory generated. If you are using [CNG](/workflow/continuous-native-generation/), then run `npx expo prebuild` to generate it.
+
+<Step label="1">
+
+### Create an upload key
```

Comment:
we might want to add a note here - if you have previously used EAS Build to build your app, you should run `eas credentials -p android` in your project and download the app signing credentials - choose the build profile and then "Download credentials from EAS to credentials.json". The **credentials.json** file keystore password, key alias, and key password, and you can use those in the following steps instead of generating as keystore.

Reply:
Thanks for the suggestion and the explanation. Decided to add a collapsible for this at the beginning of the step like this:

![CleanShot 2024-10-07 at 00 57 13@2x](https://github.com/user-attachments/assets/9c6c34dc-4727-466b-bcb8-b987942a95cb)